core/java/iesi-core/src/main/java/io/metadew/iesi/connection/database/mysql/MysqlDatabaseConnectionService.java

```diff
@@ -27,4 +27,9 @@ public String getDriver(MysqlDatabaseConnection databaseConnection) {
     public Class<MysqlDatabaseConnection> appliesTo() {
         return MysqlDatabaseConnection.class;
     }
+
+    @Override
+    public String refactorLimitAndOffset(MysqlDatabaseConnection databaseConnection, String query) {
+        return query;
+    }
```

Comment:
All XDatabaseConnectionService's extend either DatabaseConnectionService or SchemaDatabaseConnectionService (which extends DatabaseConnectionService). You could implement the default behavior in the DatabaseConnectionService class